src/ncp-spinel/SpinelNCPControlInterface.cpp

```diff
@@ -438,6 +439,20 @@ SpinelNCPControlInterface::get_name() {
 	return mNCPInstance->get_name();
 }
 
+void
+SpinelNCPControlInterface::mfg(
+    const std::string& mfg_command,
+    CallbackWithStatusArg1 cb
+) {
+	mNCPInstance->start_new_task(boost::shared_ptr<SpinelNCPTask>(
+		new SpinelNCPTaskMfg(
```

Comment:
Do this instead:

```
mNCPInstance->start_new_task(boost::shared_ptr<SpinelNCPTask>(
    new SpinelNCPTaskSendCommand(
        mNCPInstance,
        cb,
        SpinelPackData(
            SPINEL_FRAME_PACK_CMD_PROP_VALUE_SET(SPINEL_DATATYPE_UTF8_S),
            SPINEL_PROP_NEST_STREAM_MFG,
            mfg_command.c_str()
        ),
        SPINEL_DATATYPE_UTF8_S
    )
));
```

Unless `SpinelNCPTaskMfg` is doing something special that I'm not realizing.